ViewControllers/MostPopularCollectionViewController.swift

```diff
@@ -91,16 +82,34 @@ class MostPopularViewController: UIViewController, UICollectionViewDelegate {
         button.translatesAutoresizingMaskIntoConstraints = false
         button.setTitle("See more", for: .normal)
         button.titleLabel?.font = UIFont.systemFont(ofSize: 14, weight: .semibold)
-        button.tintColor = UIColor(named: "acidGreen")
+        button.tintColor = UIColor(named: "titleColor")
         button.alpha = 0
         return button
     }()
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        title = "Most popular"
-        navigationController?.navigationBar.titleTextAttributes = [.foregroundColor: UIColor.white, .font: UIFont.systemFont(ofSize: 24, weight: .heavy)]
+        setupUI()
+        
+        movieDataService.getMostPopularMoviesList { [weak self] (moviesList: [Movie]?) in
+            if let moviesList = moviesList {
+                self?.movies = moviesList
+            }
+            
+            DispatchQueue.main.async { [weak self] in
+                self?.collectionView.reloadData()
+                self?.itemInViewIndex = Int(round(Double((self?.movies.count)! / 2)))
```

Comment:
Quick note: Careful with force casting properties `!`. Unless you are sure that it will never be `nil` this can easily crash your app, I think that if you open movie details and close it super fast (before the API request is finished, the app will probably crash at the moment). We can discuss a bit more about weak/strong properties if you want 😁